### PR TITLE
remove java as a platform

### PR DESCRIPTION
**Pull request by steved**
@henders

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -106,16 +106,16 @@ GEM
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     addressable (2.3.8)
-    airbrake (4.3.4)
+    airbrake (4.3.5)
       builder
       multi_json
-    angular-rails-templates (0.2.0)
+    angular-rails-templates (0.1.4)
```
Comment (msufa) [inline Gemfile.lock]: Any particular reason to downgrade?

Comment (steved) [inline Gemfile.lock]: Hm, don't know where that came from. One sec.

**Comment by steved** [comment]
now just removing java as a platform

**Comment by msufa** [comment]
:+1:

**Pull request merged**